libs/clients/aircraft-registry/src/lib/apiConfig.ts

```diff
@@ -17,6 +17,7 @@ export const ApiConfig = {
     return new Configuration({
       fetchApi: createEnhancedFetch({
         name: 'clients-aircraft-registry',
+        organizationSlug: 'samgongustofa',
```

Comment:
In the dev meeting it was discussed that the organizationSlug would be the english slug in Contentful but perhaps we've changed our minds and decided to use the Icelandic one instead

Reply:
Un second though we have decided to have the icelandic slugs since developers recognise the Icelandic slugs better